apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.list.component.ts

```diff
@@ -12,7 +12,7 @@ import { InventoryService } from '../../../services/inventory.service';
 import { ReplaySubject, Observable, of as observableOf } from 'rxjs';
 import { Store, select } from '@ngrx/store';
 import { AppState } from '../../../store';
-import { takeUntil, take } from 'rxjs/operators';
+import { takeUntil, take, filter } from 'rxjs/operators';
```

Comment:
Please check if RxJs `filter` operator is still required, if not, then remove the unused import